src/views/AccountSettingsView.vue

```diff
@@ -407,8 +531,120 @@ const isDisabled = computed(() => {
                 <InputText id="eMail" v-model="editedUserProfile.email" disabled required />
                 <Message class="error" size="small" severity="error" variant="simple" />
               </div>
+
               <div class="input-container">
-                <label class="label" for="mobilePhoneNumber">{{ t('accountSettings.userProfile.mobilePhone') }}:</label>
+                <!-- Label for the alternative email section -->
+                <label class="label" for="alternative-eMail">{{ t('accountSettings.userProfile.alternative-email') }}:</label>
+                
+                <!-- Button to open dialog for adding alternative email -->
+                <div class="flex justify-front mt-3 mb-5">
+                  <Button
+                    label="Alternative E-Mail hinzufügen"
+                    icon="pi pi-plus"
+                    style="width: auto"
+                    :disabled="!!userProfile?.alternativeEmail"
+                    @click="visible = true"
+                  />
+                </div>
+                
+                <!-- Only show the alternative email field if one exists -->
+                <div 
+                  v-if="userProfile?.alternativeEmail" 
+                  class="flex items-center gap-1 mt-1 mb-5"
+                >
+                  <div class="alt-email-wrapper">
+                    <InputText 
+                      id="alternative-eMail"  
+                      class="alt-email-input flex-grow" 
+                      :value="userProfile?.alternativeEmail ?? ''"
+                      disabled 
+                      required 
+                    />
+                
+                    <!-- SUCCESS CHECKMARK shown after successful save -->
+                    <span
+                      v-if="altEmailSuccess"
+                      class="alt-email-icon alt-email-icon-success"
+                    >
+                      ✔
+                    </span>
+
+                    <!-- ERROR ICON shown if backend returns an error -->
+                    <span
+                      v-if="altEmailError"
+                      class="alt-email-icon alt-email-icon-error"
+                    >
+                      ✗
+                    </span>
+                  </div>
+  
+                  <!-- Trash icon deletes the existing alternative email -->
+                  <i 
+                    class="pi pi-trash alt-trash-icon cursor-pointer text-lg"
+                    @click="deleteAlternativeEmail"
+                  />
+                </div>
+              </div>
+
+              <!-- Dialog for entering the alternative email -->
+              <Dialog
+                v-model:visible="visible"
+                modal :style="{ width: '35rem' }"  
+                header="Alternative E-Mail hinzufügen"
```

Comment:
Bitte wie in Zeile 537 mit I18n implementieren.
```suggestion
                header={{"t('accountSettings.userProfile.toDo')"}}
```